packages/accounts-base/accounts_server.js

```diff
@@ -1115,6 +1115,41 @@ Ap._expireTokens = function (oldestValidDate, userId) {
   // expired tokens.
 };
 
+// Deletes expired password reset tokens from the database.
+//
+// Exported for tests. Also, the arguments are only used by
+// tests. oldestValidDate is simulate expiring tokens without waiting
+// for them to actually expire. userId is used by tests to only expire
+// tokens for the test user.
+Ap._expirePasswordResetTokens = function (oldestValidDate, userId) {
+  var tokenLiftetimeMs = this. _getPasswordResetTokenLifetimeMs();
```

Comment:
This does not look like it should work, `this. _getPasswordResetTokenLifetimeMs();` should be `this._getPasswordResetTokenLifetimeMs();`. Imho the extra space needs to be removed.